src/WebJobs.Script.SiteExtension/Compress-SiteExtension.ps1

```diff
@@ -0,0 +1,97 @@
+<#
+    .SYNOPSIS
+    Compresses the site extension.
+
+    .DESCRIPTION
+    Takes in an unzipped site extension and produces a site extension.
+
+    .PARAMETER InputPath
+    The path of the unzipped 'SiteExtension'. Leave null to scan for root in a child from here.
+
+    .PARAMETER OutputPath
+    The path to produce the site extension to. Leave null to use current directory.
+
+    .PARAMETER JitFile
+    [Array] The path of the JIT trace profiles to include in the site extension.
+
+    .PARAMETER Force
+    [Switch] Include to overwrite existing files.
+
+    .INPUTS
+    None. You can't pipe objects to Compress-SiteExtension.ps1.
+
+    .OUTPUTS
+    None. Compress-SiteExtension.ps1 doesn't generate any output.
+#>
+
+param (
+    [string] $InputPath = $null,
+    [string] $OutputPath = $null,
+    [string[]] $JitFile = @(),
+    [switch] $Force
+)
+
+if (-not $InputPath)
+{
+    $InputPath = (Get-ChildItem -Path . -Filter "extension.xml" -Recurse).Directory.FullName
+}
+
+if (Test-Path (Join-Path $InputPath "WebJobs.Script.SiteExtension.csproj"))
+{
+    Write-Error "This script should not be ran in the WebJobs.Script.SiteExtension project folder. Run this script in the root of the published site extension folder."
+    exit 1
+}
+
+if (-not (Join-Path $InputPath "extension.xml" | Test-Path))
+{
+    Write-Error "Unable to find published site extension."
+    exit 1
+}
+
+if (-not $OutputPath)
+{
+    $OutputPath = (Split-Path $InputPath -Leaf) + ".zip"
+}
+
+if (Test-Path $OutputPath)
+{
+    if ($Force)
+    {
+        Remove-Item -Path $OutputPath -Recurse -Force
+    }
+    else
+    {
+        Write-Error "OutputPath already exists. Use -Force to overwrite."
+        exit 1
+    }
+}
+
+if ($JitFile)
+{
+    $destinations = Get-ChildItem -Path $InputPath -Filter .jitmarker -Recurse
+    $JitFile | ForEach-Object {
+        $file = $_
+        $destinations | ForEach-Object {
+            Copy-Item -Path $file -Destination $_.Directory -Force
```

Comment:
Could you please add a log entry here to indicate that we are replacing the JIT trace file before compressing? For example: `Write-Host "Replacing JIT trace file in SiteExtension. Source: $file. Destination: $_.Directory"`